viennacl/src/main/scala/org/apache/mahout/viennacl/opencl/GPUMMul.scala

```diff
@@ -242,10 +242,11 @@ object GPUMMul extends MMBinaryFunc {
        N.B. This method is horribly inefficent. However there is a difference between
        getNumNonDefaultElements() and getNumNonZeroElements() which we do not always
        have access to. We created MAHOUT-1882 for this.
-         val hasElementsA = a.zSum() >  0.0
-         val hasElementsB = b.zSum() >  0.0
     */
 
+    val hasElementsA = a.zSum() >  0.0
+    val hasElementsB = b.zSum() >  0.0
+
     // A has a sparse matrix structure of unknown size.  We do not want to
```

Comment:
No problems with it thanks @andrewmusselman.  
There should be more println("gpurwrw"), etc,  in this class could you use the logger for that please?